helm/observability-bundle/templates/observability-bundle-psp-override.yaml

```
@@ -0,0 +1,29 @@
+apiVersion: v1
+data:
+  values: |
+    global:
+      rbac:
+        pspEnabled: {{ not .Values.global.podSecurityStandards.enforced }}
```

```suggestion
        pspEnabled: {{ if not .Values.global.podSecurityStandards.enforced }}
```

Is `not` alone enough ?

---

Yes it is :)

---

That actually negates the value

---

See rendered CM:

```yaml
# Source: observability-bundle/templates/observability-bundle-psp-override.yaml
apiVersion: v1
data:
  values: |
    global:
      rbac:
        pspEnabled: true
    kube-prometheus-stack:
      grafana:
        rbac:
          pspEnabled: true
      kube-state-metrics:
        podSecurityPolicy:
          enabled: true
      prometheus-node-exporter:
        rbac:
          pspEnabled: true
    prometheus-agent:
      psp:
        enabled: true
    promtail:
      rbac:
        pspEnabled: true
    # For grafana-agent
    psp:
      enabled: true
kind: ConfigMap
metadata:
  name: "-observability-bundle-psp-override"
  namespace: "default"
```

---

Oh ok :)